src/xrGame/level_script.cpp

```diff
@@ -573,6 +578,19 @@ void stop_tutorial()
         g_tutorial->Stop();
 }
 
+LPCSTR tutorial_name()
```

Comment:
Never use `LPCSTR` in the new code, use `pcstr`.

Reply:
Fixed